### PR TITLE
65% keyboard

### PR DESCRIPTION
**Pull request by Bowiten**
## Describe your project in 1-3 sentences

A 65% keyboard designed by me with support for multiple layouts.

## Hack Club Slack handle

Bowiten

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — APPROVED** [review]
@Bowiten this is soo cool! can't wait to see how this turns out :) a fabulous member of the @hackclub/bank-ops will be reaching out with next steps soon!

btw, if you're down, could you please link some of the open source keyboards that you drew inspiration from so your project proposal could potentially serve as a jumping off point for other makers looking at designing keyboards?

**Comment by Bowiten** [comment]
sure, i used kkatano's [bakeneko65](https://github.com/kkatano/bakeneko-65) and dededecline's [SST60](https://github.com/dededecline/SST60) as references, and i also used some guides such as [ai03's pcb guide](https://wiki.ai03.com/books/pcb-design) and [Masterzen's blog posts](https://www.masterzen.fr/2020/05/03/designing-a-keyboard-part-1/)

**Comment by bellesea** [comment]
> sure, i used kkatano's [bakeneko65](https://github.com/kkatano/bakeneko-65) and dededecline's [SST60](https://github.com/dededecline/SST60) as references, and i also used some guides such as [ai03's pcb guide](https://wiki.ai03.com/books/pcb-design) and [Masterzen's blog posts](https://www.masterzen.fr/2020/05/03/designing-a-keyboard-part-1/)

oh wow! that's coool :)) thank you!!

File: Bowiten.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Project Name
+bowiten65
+
+## Summary
+
+This is going to be a mechanical keyboard with a 65% layout, supporting stepped caps lock, split enter, and/or a 7u spacebar. The [PCB](https://github.com/Bowiten/bowiten65) and case are designed by me, with inspiration from a few open source keyboards. It supports MX style switches, and also has support for PCB mount stabilizers. 
+
+## Plan
+
+I plan to salvage some switches from an old keyboard by desoldering them. I'll lube them with the Krytox 205g0 that I have somewhere. I have some spare stabilizers from another keyboard that I have, which I will be using. I also plan to lube those with Krytox 205g0. After I receive the PCBs, I'll flash them with firmware that I created using [Vial](https://github.com/vial-kb/vial-qmk), a fork of [QMK](https://github.com/qmk/qmk_firmware). If everything works, then I'll solder the switches in, add stabilizers, and assemble.
```
Comment (bellesea) [inline Bowiten.md:15]:
```suggestion
I plan to salvage some switches from an old keyboard by desoldering them. I'll lube them with the Krytox 205g0 that I have somewhere. I have some spare stabilizers from another keyboard that I have, which I will be using. I also plan to lube those with Krytox 205g0. After I receive the PCBs, I'll flash them with firmware that I created using [Vial](https://github.com/vial-kb/vial-qmk), a fork of [QMK](https://github.com/qmk/qmk_firmware). If everything works, then I'll solder the switches in, add stabilizers, and assemble. I used kkatano's [bakeneko65](https://github.com/kkatano/bakeneko-65) and dededecline's [SST60](https://github.com/dededecline/SST60) as references, and i also used some guides such as [ai03's pcb guide](https://wiki.ai03.com/books/pcb-design) and [Masterzen's blog posts](https://www.masterzen.fr/2020/05/03/designing-a-keyboard-part-1/).
```

Comment (bellesea) [reply Bowiten.md:15]: cc @Bowiten what do you think about adding this in the proposal? 100% your call!

**Review by hackliv — APPROVED** [review]
Hey @Bowiten this sounds so cool! I can't wait to hear more about it. :) Dming you now on Slack about getting set up on Bank with the grant!

**Pull request merged**